source/shared/cpp/ObjectModel/Enums.h

```diff
@@ -223,7 +224,8 @@ enum class CardElementType
     TextInput,
     TimeInput,
     ToggleInput,
-    Custom
+    Custom,
+    CustomIOSDelegate,
```

Comment:
I can see how it works with _CustomIOSDelegate_, but what's the difference with the type _Custom_? Do we have code example for it for iOS?

Reply:
Custom is registering custom parser written in c++. 
Sorry, this is out dated PR. the enum has changed from CustomIOSDelegate to Unknown

---
In reply to: [172232942](https://github.com/Microsoft/AdaptiveCards/pull/1211#discussion_r172232942) [](ancestors = 172232942)